src/items.css

```diff
@@ -0,0 +1,3 @@
+.fas {
```

Comment:
There's an issue with applying the style like this that this may clash with another element where you've used the class name `fas`, this can be fixed by using [css modules](https://github.com/css-modules/css-modules), this makes the final class name have a random set of characters in it that will always be unique so if you have to classes with the same name in your code they won't have the same name in the final build so it won't matter:
<img width="325" alt="Screenshot 2020-10-12 at 01 03 37" src="https://user-images.githubusercontent.com/9036408/95693954-1fb2c700-0c27-11eb-8778-b4b2723a341a.png">

To do this we need to:

1. Rename the file.
1. Import the style as a variable
1. Use the variable to access the style properties

```diff
diff --git a/src/ListItems.js b/src/ListItems.js
index f3215bb..94a53d8 100644
--- a/src/ListItems.js
+++ b/src/ListItems.js
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import './items.css';
+import styles from './items.module.css';
 
 class ListItems extends Component {
     constructor(props) {
@@ -12,7 +12,7 @@ class ListItems extends Component {
     }
 
     createTasks(item) {
-        return <li key={item.key}>{item.text} <i class="fas fa-trash" onClick={() => this.delete(item.key)}></i></li>
+        return <li key={item.key}>{item.text} <i class={`fas fa-trash ${styles.fas}`} onClick={() => this.delete(item.key)}></i></li>
     }
 
     render() {
diff --git a/src/items.css b/src/items.module.css
similarity index 100%
-rename from src/items.css
+rename to src/items.module.css
```

Reply:
This all makes sense. I only added in the style for now just to make the cursor update when you hover over the button - which wouldn't be visible on mobile devices anyway. I can remove it, as it isn't at all important right now.